app/1.0/docs/devguide/data-binding.md

```diff
@@ -514,46 +394,49 @@ Example: { .caption }
 ```
 
 In this case, the span's `textContent` property is bound to the return value
-of `computeFullName`, which is recalculated whenever `first` or `last` changes.
+of `_formatName`, which is recalculated whenever `first` or `last` changes.
 
-### Dependent properties in computed bindings {#dependent-properties}
+**Computed bindings are one way.** A computed binding is always one-way, host-to-target.
```

Comment:
Same comment as before, should these examples that are always one way use `[[]]` to reinforce the point?